x/lockup/keeper/lock_test.go

```diff
@@ -369,47 +369,43 @@ func (suite *KeeperTestSuite) TestAddTokensToLock() {
 			expectAddTokensToLockSuccess: true,
 		},
 		{
-			name:                         "not the owner of the lock",
-			tokenToAdd:                   initialLockCoin,
-			duration:                     time.Second,
-			lockingAddress:               addr2,
-			expectAddTokensToLockSuccess: false,
```

Comment:
Am I correct in assuming you removed `expectAddTokensToLockSuccess` wherever it was false since it's not used in the test logic (because we use `else`)?

If so, I have personally found that having an expected pass or fail on every test case makes them much easier to read and understand, especially since we have no explicit standard saying "if `expectSuccess` is omitted then assume it's a failure case"

So I have a preference to keep an expected pass/fail in each test case just for clarity, although I recognize this might be a question about testing standards so this is a light preference